### ISSUE TITLE
Break test_concept_drfit_ddm into two tests

### ISSUE THREAD

**Issue by vipgupta** [uptrain-ai/uptrain#156]
Currently, we have two tests in the test file [test_concept_drfit_ddm.py](https://github.com/uptrain-ai/uptrain/blob/main/uptrain/tests/test_concept_drift_ddm.py), one for DDM and another for a custom monitor.

We want to break this file into two files that contain a test each.

This is a good first issue to get started with UpTrain.

**Comment by dineshladi** [comment]
hi @vipgupta, I have worked on this but I am not able to push my changes to the upstream branch. It throws permission denied error. Could you please add me as a contributor to this repo 
```bash
remote: Permission to uptrain-ai/uptrain.git denied to dineshladi.
fatal: unable to access 'https://github.com/uptrain-ai/uptrain.git/': The requested URL returned error: 403
```

**Comment by vipgupta** [comment]
Hi @dineshladi , You have to first fork the repository and then commit the changes there, and finally create a PR to merge. Check out the contributing instructions here: https://github.com/uptrain-ai/uptrain/blob/main/CONTRIBUTING.md

**Comment by dineshladi** [comment]
sorry, my bad. let me do that right way.

**Comment by vipgupta** [comment]
Solved by #160
